scriptmodules/ports/ikemen-go.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="ikemen-go"
+rp_module_desc="I.K.E.M.E.N GO - Clone of M.U.G.E.N to the Go programming language"
+rp_module_licence="MIT https://raw.githubusercontent.com/Windblade-GR01/Ikemen-GO/master/License.txt"
+rp_module_help="Copy characters, stages, screenpacks, etc. to $romdir/ports/ikemen-go\n\nConfig files can be found at $configdir/ports/ikemen-go/save"
+rp_module_section="exp"
+
+function depends_ikemen-go() {
+    getDepends golang-go libgl1-mesa-dev xinit xorg xorg-dev libopenal1 libopenal-dev libgtk-3-dev libasound2-dev freeglut3-dev
+}
+
+function sources_ikemen-go() {
+    gitPullOrClone "$md_build" https://github.com/Windblade-GR01/Ikemen-GO.git
```

Comment:
This should be updated to use the `rp_module_repo` variable
 See other modules for examples.

Reply:
`rp_module_repo` is used now; I didn't realize it was introduced until this comment, so thanks for letting me know!